### PR TITLE
feat(ec2): add 6xlarge InstanceSize

### PR DESCRIPTION
**Pull request by NetaNir**
closes https://github.com/aws/aws-cdk/issues/8690

----

*By submitting this pull request, I confirm that my contribution is made under the terms of the Apache-2.0 license*

### CODE REVIEW

File: packages/@aws-cdk/aws-ec2/lib/instance-types.ts [hunk 1]
```diff
@@ -391,6 +391,7 @@ export enum InstanceSize {
   XLARGE = 'xlarge',
   XLARGE2 = '2xlarge',
   XLARGE4 = '4xlarge',
+  XLARGE6 = '6xlarge',
```
Comment (shivlaks) [inline packages/@aws-cdk/aws-ec2/lib/instance-types.ts:394]: you'll need to add a linter exception for this in [package.json](https://github.com/aws/aws-cdk/blob/master/packages/%40aws-cdk/aws-ec2/package.json#L299-L300)

Comment (nija-at) [reply packages/@aws-cdk/aws-ec2/lib/instance-types.ts:394]: why not add the public api instead? In fact, would be nice to do it for all instance types here and drop the linter exceptions.

Comment (shivlaks) [reply packages/@aws-cdk/aws-ec2/lib/instance-types.ts:394]: you mean link to the API docs? - sure, I think that's more helpful to users. 
Not sure that all instance types need to be done in this PR given it's intended scope was to add the missing instance type. I'm all for removing linter exceptions. This module has the largest list of exceptions in the entire CDK (most of them the variations of windows versions)

Comment (NetaNir) [reply packages/@aws-cdk/aws-ec2/lib/instance-types.ts:394]: >I think that's more helpful to users
I would usually agree, but in this case, there isn't any real valuable docs to add to each instance size?

Comment (NetaNir) [reply packages/@aws-cdk/aws-ec2/lib/instance-types.ts:429]: Added doc to all InstanceSize

**Pull request merged**